driver.c

```diff
@@ -506,24 +506,32 @@ void drivers_init(
    {
       struct retro_system_av_info *av_info = &video_st->av_info;
       float refresh_rate                   = av_info->timing.fps;
-
+      unsigned autoswitch_refresh_rate     = settings->uints.video_autoswitch_refresh_rate;
+      bool exclusive_fullscreen            = settings->bools.video_fullscreen && !settings->bools.video_windowed_fullscreen;
+      bool windowed_fullscreen             = settings->bools.video_fullscreen && settings->bools.video_windowed_fullscreen;
+      bool all_fullscreen                  = settings->bools.video_fullscreen || settings->bools.video_windowed_fullscreen;
+   
       if (  refresh_rate > 0.0 &&
             !settings->uints.crt_switch_resolution &&
             !settings->bools.vrr_runloop_enable &&
-            !settings->bools.video_windowed_fullscreen &&
-            settings->bools.video_fullscreen &&
+            autoswitch_refresh_rate != AUTOSWITCH_REFRESH_RATE_OFF &&

```

Comment:
Since `autoswitch_refresh_rate` is enabled by default and the option to disable it is hidden behind a `video_display_server_has_resolution_list()`, can we force-disable it with the same guard here? i.e. just make that:

```c
...
!settings->bools.vrr_runloop_enable &&
video_display_server_has_resolution_list() &&
(autoswitch_refresh_rate != AUTOSWITCH_REFRESH_RATE_OFF) &&
...

Reply:
All done!